### DISCUSSION TITLE
Media Queries Not Based On Screen Width

### DISCUSSION THREAD

**Discussion by armordog** [chakra-ui/chakra-ui#4690] [Q&A] [answered]
There is a LOT of support in Chakra for width-based media queries.

I would like to add a `max-resolution` media query but I cannot see how to specify arbitrary CSS in a Chakra theme.

Something like
```
@media (max-resolution: 2dppx) {
  body {
    -webkit-font-smoothing: antialiased;
  }
}
```

**Comment by primos63** [comment, marked as answer]
Assuming this is a global setting, add it to your global styles

```javascript
import { extendTheme } from "@chakra-ui/react";

export const theme = extendTheme({
  styles: {
    global: {
      "@media (max-resolution: 2dppx)": {
        body: {
          "-webkit-font-smoothing": "antialiased"
        }
      }
    }
  }
});
```

**Reply by armordog** [reply]
@primos63 thank you so much!
I can never figure out exactly what I'm supposed to put in `extendTheme`.
